.github/actions/brew/action.yml

```diff
@@ -31,4 +31,4 @@ runs:
           libxml2 \
           libxslt \
           postgresql
-        brew link icu4c gettext --force
+        brew link bison icu4c gettext --force
```

Comment:
needed for proper Bison 3.x install, see https://stackoverflow.com/questions/24835116/bison-latest-version-installed-but-not-in-use#29053701